photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -71,6 +69,8 @@ final class Driver(val params: GameParams, val sparkContext: SparkContext, val l
       data
     }
 
+    logger.info(s"Time elapsed after computing features summary: ${timer.durationSeconds} (s)\n")
```

Comment:
This seems out of place in this version of the code. Did you mean to commit this line?